web-app/react-app/components/AuthPage/pages/RegisterPage/index.jsx

```diff
@@ -0,0 +1,204 @@
+                    </Actions>
+                </AuthStep>
+                <AuthStep title="Sign up" showBack>
+                    <Input
+                        name="password"
+                        type="password"
+                        label="Create a Password"
+                        value={form.password}
+                        onChange={onInputChange(setFormField)}
+                        error={errors.password}
+                        processing={step === 1 && isProcessing}
+                        autoComplete="new-password"
+                        measureStrength
+                        autoFocus
+                    />
+                    <Actions>
+                        <Button disabled={isProcessing}>Next</Button>
+                    </Actions>
+                </AuthStep>
+                <AuthStep title="Sign up" showBack>
+                    <Input
+                        name="confirmPassword"
+                        type="password"
+                        label="Confirm your password"
+                        value={form.confirmPassword}
+                        onChange={onInputChange(setFormField)}
+                        error={errors.confirmPassword}
+                        processing={step === 2 && isProcessing}
+                        autoComplete="new-password"
+                        autoFocus
+                    />
+                    <Actions>
+                        <Button disabled={isProcessing}>Next</Button>
+                    </Actions>
+                </AuthStep>
+                <AuthStep title="Timezone" showBack>
+                    <Select
+                        name="timezone"
+                        value={form.timezone}
+                        options={moment.tz.names().map(tz => ({
+                            value: tz,
+                            label: tz,
+                        }))}
+                        onChange={this.onTimezoneChange}
+                        autoFocus
+                    />
+                    <Actions>
+                        <Button disabled={isProcessing}>Next</Button>
+                    </Actions>
+                </AuthStep>
+                <AuthStep
+                    title="Terms"
+                    onSubmit={this.submit}
+                    onSuccess={redirect}
+                    onFailure={this.onFailure}
+                    showBack
+                >
+                    <div className={cx(authPanelStyles.spaceMedium, authPanelStyles.centered)}>
+                        <Checkbox
+                            name="toc"
+                            checked={form.toc}
+                            onChange={onInputChange(setFormField)}
+                            error={errors.toc}
+                            autoFocus
+                        >
+                            I agree with the <a href="#">terms and conditions</a>, and <a href="#">privacy policy</a>.
```

Comment:
`<a href="https://s3.amazonaws.com/streamr-public/streamr-terms-of-use.pdf" target="_blank">terms and conditions</a>`

`<a href="https://s3.amazonaws.com/streamr-public/streamr-privacy-policy.pdf" target="_blank" >privacy policy</a>`

Reply:
Good point! Thx.
Probably will just use the same kind of `links.js` file there as well

Reply:
And big thanks for the links!